numpy/core/_multiarray_umath.py

```diff
@@ -8,6 +8,12 @@
     if isinstance(attr, ufunc):
         globals()[item] = attr
 
+# The NumPy 1.x import_array() and import_ufunc() mechanisms expect
+# these symbols to be available without error. If an extension was compiled
+# against NumPy 1.x it will have these paths hard-coded.
+globals()['_ARRAY_API'] = _multiarray_umath._ARRAY_API
+globals()['_UFUNC_API'] = _multiarray_umath._UFUNC_API
```

Comment:
```suggestion
_ARRAY_API = _multiarray_umath._ARRAY_API
_UFUNC_API = _multiarray_umath._UFUNC_API
```
No point in globals, no?

We need to break 1.x opening here soon enough, but I thought we kept the path in place also for 2.x.